packages/website/src/search/componentTitleList.csv

```diff
@@ -0,0 +1,98 @@
+Iconography
```

Comment:
Why this file?

Reply:
It's to feed ML for query suggestions :see_no_evil: